src/common/personinfo.js

```diff
@@ -167,8 +167,34 @@ let personinfo = function (tour, aConfig) {
   });
 };
 
+
+const PERSISTENT_LOCALSTORAGE_KEY="localStorageUsed";
+let tryLocalStorage = function (store=localStorage) {
+  try {
+    var haveChecked = JSON.parse(store.getItem(PERSISTENT_LOCALSTORAGE_KEY) || 0);
+    if (!haveChecked || haveChecked < 2) {
+      store.setItem(PERSISTENT_LOCALSTORAGE_KEY, ++haveChecked);
+      return;
+    }
+  } catch (e) {
+    store.setItem(PERSISTENT_LOCALSTORAGE_KEY, 1);
```

Comment:
claim: it would throw if there was not-JSON in that field.  I don't know how that would happen.  User intervention.  I don't think it should kill the whole recipe.  I am open to being told otherwise.